src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,8 +121,11 @@ public static Class parseClass(String classDatetime) throws ParseException {
         LocalDate date = parseDate(datetimeStr);
         LocalTime startTime = parseTime(startTimeStr);
         LocalTime endTime = parseTime(endTimeStr);
+        if (endTime.isBefore(startTime) || endTime.equals(startTime)) {
+            throw new ParseException(Class.INVALID_DURATION_ERROR_MESSAGE);
```

Comment:
Test cases should be added to cover such situation. You can find similar test cases in `~/tp/src/test/java/seedu/address/logic/parser/ParserUtilTest.java`.